account_operating_unit/tests/test_payment_operating_unit.py

```diff
@@ -76,7 +76,7 @@ def test_payment_from_two_invoices(self):
         for payment in payments:
             # Validate that inter OU balance move lines are created
             self.assertEqual(len(payment.move_id.line_ids), 2)
-            self.assertAlmostEqual(payment.amount, 115000)
+            self.assertEqual(payment.amount, invoices[0].amount_total)
```

Comment:
No risk of float precision issue ?

Reply:
I don't think so. Since the decimal precision of the amount field is based on the currency, we should not have any difference between the amount of the invoice and the amount of the payment.